### PR TITLE
Bug 1119278 - Intermittent test_music_change_rating.py

### PR DESCRIPTION
**Pull request by chirarobert**

### CODE REVIEW

**Comment by try-server-hook** [comment]
[chirarobert chirarobert](https://github.com/chirarobert) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=a9417851474d)

File: tests/python/gaia-ui-tests/gaiatest/apps/music/regions/player_view.py [hunk 1]
```diff
@@ -32,7 +30,7 @@ def tap_cover_in_player_view(self):
             Wait(self.marionette).until(expected.element_present(*self._rating_view_locator))))
 
     def _get_star_locator(self, rating):
-        return (By.CSS_SELECTOR, '.rating-star[data-rating="%s"]' % rating)
+        return By.CSS_SELECTOR, '.rating-star[data-rating="%s"]' % rating
```
Comment (chirarobert) [inline tests/python/gaia-ui-tests/gaiatest/apps/music/regions/player_view.py]: The parenthesis are not required to return tuples.

File: tests/python/gaia-ui-tests/gaiatest/tests/functional/music/test_music_change_rating.py [hunk 2]
```diff
@@ -45,4 +45,3 @@ def test_music_change_rating(self):
         #change the rating to 1
         player_view.tap_star(1)
         self.assertEquals(player_view.star_rating, 1)
-
```
Comment (chirarobert) [inline tests/python/gaia-ui-tests/gaiatest/tests/functional/music/test_music_change_rating.py]: I believe the no new line at end of file error is wrong. In the editor the is a new line.
What I changed here is that I removed some white space from the line.

File: tests/python/gaia-ui-tests/gaiatest/apps/music/regions/player_view.py (hunk 1 above)
Comment (bebef1987) [inline tests/python/gaia-ui-tests/gaiatest/apps/music/regions/player_view.py]: I still think this is requierd. We need to make sure we return a tuple from this method.
https://github.com/chirarobert/gaia/blob/bug_1119278/tests/python/gaia-ui-tests/gaiatest/apps/music/regions/player_view.py#L41

**Comment by try-server-hook** [comment]
[chirarobert chirarobert](https://github.com/chirarobert) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=7d4fb4fb0549)

**Pull request merged**